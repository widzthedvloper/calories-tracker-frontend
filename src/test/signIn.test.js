import React from 'react';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';

it('renders button correctly', () => {
    const tree = renderer
    .create(<button>Sign In</button>)
    .toJSON();
    expect(tree).toMatchSnapshot();
})

it('renders link correctly', () => {
    const tree = renderer
    .create(<a>Create a new account</a>)
    .toJSON();
    expect(tree).toMatchSnapshot();
})

it('renders label correctly', () => {
    const tree = renderer
    .create(<label>Email</label>)
    .toJSON();
    expect(tree).toMatchSnapshot();
})

it('renders button correctly', () => {
    const tree = renderer
    .create(<button>Sign Up</button>)
    .toJSON();
    expect(tree).toMatchSnapshot();
})

it('renders link correctly', () => {
    const tree = renderer
    .create(<a>Already have an account? Sign In!</a>)
    .toJSON();
    expect(tree).toMatchSnapshot();
})

it('renders title correctly', () => {
    const tree = renderer
    .create(<h1>Calories Tracker</h1>)
    .toJSON();
    expect(tree).toMatchSnapshot();
})

it('renders link correctly', () => {
    const tree = renderer
    .create(<a>Add Food</a>)
    .toJSON();
    expect(tree).toMatchSnapshot();
})

it('renders link correctly', () => {
    const tree = renderer
    .create(<a>Dash</a>)
    .toJSON();
    expect(tree).toMatchSnapshot();
})

it('renders link correctly', () => {
    const tree = renderer
    .create(<a>Ingredient</a>)
    .toJSON();
    expect(tree).toMatchSnapshot();
})

it('renders link correctly', () => {
    const tree = renderer
    .create(<a>More</a>)
    .toJSON();
    expect(tree).toMatchSnapshot();
})
