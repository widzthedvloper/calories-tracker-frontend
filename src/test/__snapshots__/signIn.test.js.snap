// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders butto correctly 1`] = `
<button>
  Sign In
</button>
`;

exports[`renders button correctly 1`] = `
<button>
  Sign In
</button>
`;

exports[`renders button correctly 2`] = `
<button>
  Sign Up
</button>
`;

exports[`renders label correctly 1`] = `
<label>
  Email
</label>
`;

exports[`renders link correctly 1`] = `
<a>
  Create a new account
</a>
`;

exports[`renders link correctly 2`] = `
<a>
  Already have an account? Sign In!
</a>
`;

exports[`renders link correctly 3`] = `
<a>
  Add Food
</a>
`;

exports[`renders link correctly 4`] = `
<a>
  Dash
</a>
`;

exports[`renders link correctly 5`] = `
<a>
  Ingredient
</a>
`;

exports[`renders link correctly 6`] = `
<a>
  More
</a>
`;

exports[`renders title correctly 1`] = `
<h1>
  Calories Tracker
</h1>
`;
